package.json

```diff
@@ -69,9 +74,12 @@
     "flow-copy-source": "^1.3.0",
     "jest": "^22.4.0",
     "koa": "^2.5.0",
-    "rimraf": "^2.6.1"
+    "rimraf": "^2.6.1",
+    "typescript": "^2.7.2",
+    "typescript-eslint-parser": "^14.0.0"
   },
   "peerDependencies": {
+    "@types/koa": "2.x",
```

Comment:
We're adding this as a peer dependency in case somebody is using typescript. Our type definitions require the koa type definitions to exist, but if you're not using typescript this will just give a warning which could be ignored. Not sure if there's a better approach here.